src/docs/guides/laravel.md

```diff
@@ -0,0 +1,185 @@
+    - The CLI will now scan, compress and upload our Laravel app files to Railway's backend for deployment.
+
+## Deploy via Custom Scripts
+
+This setup deploys your Laravel app on Railway, ensuring that your database, scheduled tasks (crons), and queue workers are all fully operational.
+
+The deployment structure follows a "majestic monolith" architecture, where the entire Laravel app is managed as a single codebase but split into four separate services on Railway:
+- **App Service**: Handles HTTP requests and user interactions.
+- **Cron Service**: Manages scheduled tasks (e.g., sending emails or running reports).
+- **Worker Service**: Processes background jobs from the queue.
+- **Database Service**: Stores and retrieves your application's data.
+
+<Image src="https://res.cloudinary.com/railway/image/upload/f_auto,q_auto/v1727910244/docs/quick-start/deploy%20architecture.png"
+alt="screenshot of the deploy architecture of the Laravel app"
+layout="responsive"
+width={3118} height={1776} quality={100} />
+_My Majestic Monolith Laravel app_
+
+Please follow these steps:
+
+1. Create three bash script files in the root directory of your Laravel app: `run-app.sh`, `run-worker.sh`, and `run-cron.sh`. 
+
+    After creating the files, make them executable by running `chmod +x` on each file in the terminal. This will allow your scripts to run properly when executed.
+    - Add the content below to the `run-app.sh` file:
+        ```bash
+            #!/bin/bash
+
+            # Make sure this file has executable permissions, run `chmod +x run-app.sh`
+
+            # Build assets using NPM
+            npm run build
+
+            # Clear cache
+            php artisan optimize:clear
+
+            # Cache the various components of the Laravel application
+            php artisan config:cache
+            php artisan event:cache
+            php artisan route:cache
+            php artisan view:cache
+
+            # Run any database migrations
+            php artisan migrate --force
+        ```
+    -  Add the content below to the `run-worker.sh` file:
+        ```bash
+            #!/usr/bin/env bash
+
+            # Make sure this file has executable permissions, run `chmod +x run-worker.sh`
+
+            # Run the queue worker
+            php artisan queue:work     
+        ```
+    -  Add the content below to the `run-cron.sh` file:
+        ```bash
+            #!/usr/bin/env bash
+
+            # Make sure this file has executable permissions, run `chmod +x run-cron.sh`
+
+            # Runs the Laravel scheduler every minute
+            while [ true ]
+                do
+                    echo "Running the scheduler..."
+                    php artisan schedule:run --verbose --no-interaction &
+                    sleep 60
+                done
+        ```
+2. Create a Database service on the <a href="/overview/the-basics#project--project-canvas" target="_blank">Project Canvas.</a>
+     - Click on **Deploy**.
```

Comment:
What type fo database service? I think this is a bit unclear

Reply:
Good point!

Reply:
Fixed this